js/flightlog_fields_presenter.js

```diff
@@ -1023,6 +1023,97 @@ function FlightLogFieldPresenter() {
             'debug[6]':'Not Used',
             'debug[7]':'Not Used',
         },
+        'FAILSAFE' : {
+            'debug[all]': 'GPS Dilution of Precision',
+            'debug[0]': 'Number of Satellites',
+            'debug[1]': 'pDOP (positional - 3D)',
+            'debug[2]': 'hDOP (horizontal - 2D)',
+            'debug[3]': 'vDOP (vertical - 1D)',
+        },
+        'GYRO_CALIBRATION' : {
+            'debug[all]': 'GYRO_CALIBRATION',
+            'debug[0]': 'GYRO_CALIBRATION1',
+            'debug[1]': 'GYRO_CALIBRATION2',
+            'debug[2]': 'GYRO_CALIBRATION3',
+            'debug[3]': 'GYRO_CALIBRATION4',
+        },
+        'ANGLE_MODE' : {
+            'debug[all]': 'ANGLE_MODE',
+            'debug[0]': 'ANGLE_MODE1',
+            'debug[1]': 'ANGLE_MODE2',
+            'debug[2]': 'ANGLE_MODE3',
+            'debug[3]': 'ANGLE_MODE4',
+        },
+        'ANGLE_TARGET' : {
+            'debug[all]': 'ANGLE_TARGET',
+            'debug[0]': 'ANGLE_TARGET1',
+            'debug[1]': 'ANGLE_TARGET2',
+            'debug[2]': 'ANGLE_TARGET3',
+            'debug[3]': 'ANGLE_TARGET4',
+        },
+        'CURRENT_ANGLE' : {
+            'debug[all]': 'CURRENT_ANGLE',
+            'debug[0]': 'CURRENT_ANGLE1',
+            'debug[1]': 'CURRENT_ANGLE2',
+            'debug[2]': 'CURRENT_ANGLE3',
+            'debug[3]': 'CURRENT_ANGLE4',
+        },
```

Comment:
Let's add missing items in separate PR to keep intent of current PR isolated.

Reply:
resolved in current codebase